app/models/service_contract.rb

```diff
@@ -28,11 +28,11 @@ def self.provided_by(account)
 
   include ThreeScale::Search::Scopes
 
-  self.allowed_sort_columns = %w{ cinstances.state accounts.org_name cinstances.created_at } # can't order by plans.name, service.name - mysql blows up
+  self.allowed_sort_columns = %w{ cinstances.state accounts.org_name cinstances.created_at plans.name } # can't order by plans.name, service.name - mysql blows up
```

Comment:
Sorting by plan was broken - the column got the hightlight, but the table data was not updated - the SQL query wasn't including this sorting option.

Reply:
maybe update the comment also. Because it claims what you enabled will crash MySQL and that is hopefully not true :)

Reply:
Hm, I am wondering now - maybe it does indeed blow up? :thinking: 

Maybe it's not visible in test because there is little data?

But then, if we wanted to disable sorting by plans, the column shouldn't appear sortable in the UI.
Do you think it's safer to just make it unsortable?

I can keep the test (commented out) so we can enable sorting in another PR...